src/screens/home/HomeScreen.tsx

```diff
@@ -90,36 +70,77 @@ const Content = () => {
         case SystemStatus.BluetoothOff:
           return <BluetoothDisabledView />;
         case SystemStatus.Active:
-        case SystemStatus.Unknown:
           return <NoExposureView />;
+        default:
+          return null;
       }
```

Comment:
Is this required? Shouldn't we always default to an actual view? Not sure having a blank screen is the best fallback here.

Reply:
It's currently flashing from INACTIVE to ACTIVE. So, this is a fix for it.

Reply:
Shouldn't we wait for the start function to complete then before rendering? I had thought UNKNOWN was a valid state

Reply:
When I start the app in ios simulator it just shows the blank screen, I think we need to relook at a better way to fix this.
![Screen Shot 2020-06-23 at 10 56 19 AM](https://user-images.githubusercontent.com/58042767/85419558-3f22c880-b540-11ea-8a7b-a378d6a5d9c7.png)

Reply:
That's what I was suggesting in my previous attempt. cc @gavrix

Reply:
this content is only rendering text inside home, not overlay. That content for unknown state is, well, unknown. if not rendering nothing, we could literally show "Status known" or show activity indicator.

Reply:
empty bottom sheet contents is a separate problem not related to this.